web/beacon-app/package.json

```diff
@@ -30,6 +30,8 @@
     "watch:css": "npx tailwindcss -o ./public/tailwind.css --watch",
     "precommit": "lint-staged",
     "cypress": "cypress open --e2e --browser chrome",
+    "cypress:report": "cypress run --e2e --browser chrome --reporter mochawesome --reporter-options reportDir=cypress/reports,overwrite=false,html=false,json=true",
```

Comment:
I didn't see a report after running this command. I looked at how Cypress reports was set up in another repo and some `mochawesome` packages should be installed as `devDependencies`: https://github.com/trisacrypto/directory/blob/main/web/gds-user-ui/package.json#L183-L185

I also noticed that another script was added here: https://github.com/trisacrypto/directory/blob/main/web/gds-user-ui/package.json#L77-L78

It also looks like the `.gitignore` was updated so that the `mochawesome` JSON file isn't committed: https://github.com/trisacrypto/directory/blob/main/web/gds-user-ui/.gitignore#L10-L13

Reply:
Oops, my mistake! I removed it from the wrong spot when opening the PR. Thanks for catching that!

Reply:
The problem stems from the missing of the `mochawesome` dependency and the necessary configuration in ".babelrc" for the Cypress e2e instrumentation.